content/en/resources/services.md

```diff
@@ -84,7 +84,7 @@ For more information, see the Kubernetes [GitHub Repository Guidelines].
 
 ### Contributor communications
 
-Contributor communication requests, such as tweets ([@k8contributors]), blog
+Contributor communication requests, such as Tweets from the [K8sContributors Twitter account](https://twitter.com/K8sContributors), blog
```

Comment:
Hey, @sandipanpanda overall this PR looks good. However, I think it would be better to keep the link text as `@K8sContributors` instead of `K8sContributors Twitter account`.

Like:
**tweets from \[@K8sContributors\](https://twitter.com/K8sContributors)**